doozerlib/rpmcfg.py

```diff
@@ -233,6 +234,25 @@ def get_package_name(self, default=-1):
 
         raise IOError(f'Unable to find Name: field in rpm spec: {spec_path}')
 
+    def get_package_name(self, default=-1):
+        """
+        Returns the brew package name for the distgit repository. Package names for RPMs
+        do not necessarily match the distgit component name. The package name is controlled
+        by the RPM's spec file. The 'name' field in the doozer metadata should match what is
+        in the RPM's spec file.
+        :param default: If specified, this value will be returned if package name could not be found.
+        :return: The package name - Otherwise, the specified default.
+                If default is not passed in, an exception will be raised if the package name can't be found.
+        """
+        package_name = self.config.name  # This should be the package name for the RPM
+        if package_name:
+            return package_name
+
+        if default != -1:
```

Comment:
this looks weird / non-pythonic... default ought to be `None` and this
```suggestion
        if default is not None:
```
but i guess this is everywhere and we can fix that some other time when it annoys me :)